docs/guide/function/depend.md

```diff
@@ -4,11 +4,53 @@ title: 依赖管理
 
 # {{ $frontmatter.title }}
 
-依赖管理可以为应用增加第三方包模块，可以理解为nodejs的`npm install`
+应用开发过程中，通常有添加 `npm` 依赖的需求，`laf` 提供了在线可视化管理这些第三方包模块的方法, 用户可非常方便的进行搜索、安装、升级和卸载。
+
+## 添加依赖
+
+![](https://oss.lafyun.com/bstqua-images/add-packages.png)
```

Comment:
Can you put the image in the md file directory instead of using the external image now

Reply:
ok